src/composables/use-pages.ts

```diff
@@ -7,26 +7,32 @@ export default function usePages(isNew = false) {
     ? [
         {
           id: 'about',
+          name: 'navigation.about',
```

Comment:
There is a `i18n:check` script in package.json that checks for unused and missing i18n keys. Unfortunately, it can't understand dynamic keys, so it reports the key as missing if it's dynamic. This means that we cannot use it in the CI. If we convert all dynamic keys to static keys, we will be able to check the i18n for completeness in the CI, @dhruvkb

Reply:
Ok, makes sense.